tests/e2e/lib.sh

```diff
@@ -353,10 +361,20 @@ deploy_sensor_via_operator() {
        die "COLLECTION_METHOD not set"
     fi
 
+    if [[ "${SENSOR_SCANNER_SUPPORT:-}" == "true" ]]; then
```

Comment:
Is there a case when it equals to false?

Reply:
Generally speaking, yes.
That switch controls the activation of the local scanning feature.

Reply:
I find it a little confusing, could you please add `LOCAL_` to the en?

Reply:
Well, two things: since I haven't introduced this variable I would prefer to keep the scope of this change small. Furthermore, and more importantly, such a renaming could potentially break other engineer's workflows.